packages/picasso/src/AvatarUpload/styles.ts

```diff
@@ -0,0 +1,131 @@
+import { SizeType } from '@toptal/picasso-shared'
+import { createStyles, Theme } from '@material-ui/core/styles'
+
+const SETTINGS = {
+  small: {
+    dimensions: 5,
+    cornerSize: 1,
+    hipotenusa: Math.sqrt(2) * 1,
+    cornerCoordinate: 0.65,
+  },
+  large: {
+    dimensions: 10,
+    cornerSize: 1.5,
+    hipotenusa: Math.sqrt(2) * 1.5,
```

Comment:
I'm a little bit confused by this calculation. The hypotenuse is calculated `c² = a² + a²`. I would maybe move this calculation to a separate function
```ts
// c² = a² + a²
const getHypotenuse = (pendant: number) => Math.sqrt(Math.pow(pendant) * 2)
```

```suggestion
    hypotenuse: getHypotenuse(1.5),
```